frontend/templates/views/partials/pixi/status-intro.j2

```diff
@@ -19,13 +19,12 @@
         {{ pixi.staticText.statusIntro.loadingCopy|replace('${finishedChecks}', '<span id="finished-checks">0</span>')|replace('${totalChecks}','<span id="total-checks">0</span>')|safe }}
       </p>
       <div class="ap-m-pixi-status-intro-banner-ctas">
-        <button class="ap-a-btn"
-            on="tap:recommendations.scrollTo(duration=400)"
-            type="button"
-            name="button">
+        <a href="#recommendations" class="ap-a-btn"
```

Comment:
Can we keep `on="tap:recommendations.scrollTo(duration=400)"`?

Reply:
Unfortunately even with that there it will jump to the anchor tag. Another option though is to add `scroll-behavior: smooth;` on the document CSS, but I believe this has to be on the outer document and I am not sure how to add this for just the pixi page.